src/lista/repeticao/Ex92.java

```diff
@@ -1,2 +1,70 @@
-package lista.repeticao;public class Ex92 {
+package lista.repeticao;
+
+import java.util.Scanner;
+
+public class Ex92 {
+    public static void main(String[] args) {
+        Scanner leia = new Scanner(System.in);
+        int i;
+        int total_alunos, alunos_maior_renda = 0;
+        double porcentagem_despesas = 0, contador_depesas = 0, porcentagem_alimentacao = 0, soma_alimentacao = 0, alimentacao_despesas = 0;
+        double porcentagens_pessoal=0, porcentagem_familiar=0;
+        System.out.println("Digite o total de alunos: ");
+        total_alunos = leia.nextInt();
+        double total_gasto_outras_despesas[] = new double[total_alunos];
+        for (i = 0; i < total_alunos; i++) {
+            System.out.println(" Total gasto com outras despesas: ");
+            total_gasto_outras_despesas[i] = leia.nextDouble();
+        }
+        for (i = 0; i < total_alunos; i++) {
+            if (total_gasto_outras_despesas[i] > 200) {
+                contador_depesas++;
+            }
+        }
+
+        double renda_familiar[] = new double[total_alunos];
+        for (i = 0; i < total_alunos; i++) {
+            System.out.println(" Renda familiar: ");
+            renda_familiar[i] = leia.nextDouble();
+        }
+        double renda_pessoal[] = new double[total_alunos];
+        for (i = 0; i < total_alunos; i++) {
```

Comment:
O mais correto, seria usar o "while" ou "do while" para que o programa pare quando for digitado uma renda pessoal igual à zero, e não precisa de todos esses "for".